assets/helpers/url.js

```diff
@@ -2,14 +2,41 @@
 
 // ----- Functions ----- //
 
-const getQueryParameter = (paramName: string, defaultValue: string): string => {
+const getQueryParameter = (paramName: string, defaultValue?: string): ?string => {
 
   const params = new URLSearchParams(window.location.search);
 
   return params.get(paramName) || defaultValue;
 
 };
 
-// ----- Exports ----- //
 
-export default getQueryParameter;
+const addQueryParamToURL = (str: string, paramsKey: string, paramsValue: ?string): string => {
```

Comment:
Good catch @rupertbates!! I renamed the variable in 98a48d2bc7b68f64869568aabcba29b55fa11395